INSTALL.md

```diff
@@ -305,6 +305,20 @@ Then proceed to [Choosing where to store pokeemerald (Linux)](#choosing-where-to
 >   then you will have to install devkitARM. Install all the above packages except binutils-arm-none-eabi, and follow the instructions to
 >   [install devkitARM on Debian/Ubuntu-based distributions](#installing-devkitarm-on-debianubuntu-based-distributions).
 </details>
+    
+### Arch Linux
+Run this command as root to install the necessary packages:
+```console
+# pacman -S base-devel arm-none-eabi-binutils git
```

Comment:
Exclude the `#` here and below, beginners will use these instructions and likely include it if they're copy-pasting

Reply:
D'aw, I thought GitHub would do the reasonable thing and mark the `console` prompt as un-selectable. --'